main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Projects/NewProjectController.cs

```diff
@@ -613,14 +613,15 @@ public void MoveToPreviousPage ()
 
 		public async Task Create ()
 		{
+			OnProjectCreating ();
```

Comment:
@mrward @mykyta-bondarenko in case we want to ensure it's called on the UI thread:
```suggestion
			Runtime.AssertMainThread ();
			OnProjectCreating ();
```